assets/script.js

```diff
@@ -82,60 +84,41 @@ function startGame(){
     createQuestions();
 };
 
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf
+function getIndexFromId(id) {
+    var ids = ["buttonA", "buttonB", "buttonC", "buttonD"];
+    return ids.indexOf(id);
+}
+
+function checkAnswer(event) {
+    var userAnswer = getIndexFromId(event.target.id);
+    if (userAnswer === currentQuestion.answer) {
+        score++
+    } else {
+        timeLeft = timeLeft - 10;
+    }
+}
+
```

Comment:
using a function to check the answer, instead of re-writing it multiple times. This is where making `currentQuestion` is useful, because it can be accessed in this function now. This makes the code less repetitive. If you need to change this function, you only have to do it in one place now.